controllers/gce/storage/configmaps.go

```diff
@@ -50,46 +46,59 @@ type ConfigMapVault struct {
 // If this method returns an error, it's guaranteed to be apiserver flake.
 // If the error is a not found error it sets the boolean to false and
 // returns and error of nil instead.
-func (c *ConfigMapVault) Get() (string, bool, error) {
+func (c *ConfigMapVault) Get() (map[string]string, bool, error) {
```

Comment:
Thanks for the example -- I think the summary is just that we'll be issuing more configmap reads but won't be exposing the map which I think per go is fundamentally not threadsafe.